server/middleware/cache-control.js

```diff
@@ -16,22 +13,13 @@ export default defineEventHandler((event) => {
 
     const isIpx        = pathname.match(/\/_ipx\//)
     const isAsset      = isIpx || pathname.match(/(.+)\.(avif|webp|jpg|jpeg|gif|css|png|js|ico|svg|mjs)/)
-    const isContextApi = pathname.match(/\/api\/context\//);
-    
-    const isListsApi   = pathname.match(/\/api\/list\//);
-    const isApi        = pathname.match(/\/api\//);
-    const isPageApi    = pathname.match(/\/api\/page\//);
     const isNuxt       = pathname.match(/\/_nuxt\//);
 
 
     if(isNoCache)
         res.setHeader('Cache-Control', `no-store, max-age=0`);
     else if(isAsset || isNuxt)
         res.setHeader('Cache-Control', `max-age=${year}, stale-if-error=${week}`);
-    else if(!isPageApi && (isContextApi || isListsApi ||isApi))
-        res.setHeader('Cache-Control', `max-age=${day}, stale-if-error=${week}, stale-while-revalidate=${day}`);
-    else if(isPageApi ||  isMenusApi )
-        res.setHeader('Cache-Control', `max-age=${min}, stale-if-error=${week}, stale-while-revalidate=${day}`);
     else
-        res.setHeader('Cache-Control', `max-age=${min}, stale-if-error=${week}, stale-while-revalidate=${day}`);
+        res.setHeader('Cache-Control', `max-age=15, stale-if-error=${week}, stale-while-revalidate=${day}`);
```

Comment:
I guess those number shoudl come from env

Reply:
well yes and no.  year, week and day do not change.  however the type of files and how long should.   There have been tweaks to this file later addressing different types.

its a good idea but i want to osleep on th eimplimentation and get main upto date first.  i will make a card. BL-682